simplq/src/styles/sidePanel.module.scss

```diff
@@ -1,10 +1,12 @@
 @import 'utils';
 
 .side-panel {
-  padding-top: 1.5rem;
-  padding-right: 6rem;
+  box-sizing: border-box;
+  padding: 0 6rem 39px 0;
+  display: flex;
+  flex-flow: column;
 
```

Comment:
Could you try replacing px with rem? We're using rem consistently everywhere else.

Reply:
Done